spec/forms/training_details_form_spec.rb

```diff
@@ -48,8 +48,8 @@
         end
       end
 
-      context "duplicate" do
-        let!(:existing_trainee) { create(:trainee, trainee_id: "Test123") }
+      context "duplicate active trainee" do
+        let!(:existing_trainee) { create(:trainee, trainee_id: "Test123", state: "recommended_for_award") }
```

Comment:
It appears that adding the state doesn't make a difference to the outcome of the test because they both have the same `trainee_id`.